### PR TITLE
Feature/dp 542 adding an exclusion

### PR DESCRIPTION
**Pull request by JBaigGoaco**

### CODE REVIEW

**Review by dharmverma — APPROVED** [review]

**Review by dpatel017 — APPROVED** [review]

File: Frontend/CO.CDP.OrganisationApp/Pages/Forms/_FormElementMultiLineInput.cshtml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+@{
+    var hasError = ((TagBuilder)Html.ValidationMessageFor(m => m.TextInput)).HasInnerHtml;
+}
+
+<div class="govuk-form-group">
+
+    @if (!string.IsNullOrWhiteSpace(Model.Heading))
+    {
+        <label class="govuk-label govuk-label--l" for="declarationInput">@Model.Heading</label>
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Forms/_FormElementMultiLineInput.cshtml:11]: Is this label supposed to be associated with the textarea below? The `for` on the label needs to match an `id` on something.

File: Frontend/CO.CDP.OrganisationApp/Pages/Forms/_FormElementTextInput.cshtml [hunk 2]
```diff
@@ -5,15 +5,15 @@
...
 <div class="govuk-form-group">
-    @if (!string.IsNullOrWhiteSpace(Model.Caption))
-    {
-        <span class="govuk-caption-l">@Model.Caption</span>
-    }
-
+   
     @if (!string.IsNullOrWhiteSpace(Model.Heading))
     {
         <label class="govuk-label govuk-label--l" for="declarationInput">@Model.Heading</label>
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Forms/_FormElementTextInput.cshtml:11]: Same as above - the label is not associated with anything

**Review by andymantell — CHANGES_REQUESTED** [review]

File: Frontend/CO.CDP.OrganisationApp/Pages/Forms/_FormElementMultiLineInput.cshtml (hunk 1 above)
Comment (JBaigGoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Forms/_FormElementMultiLineInput.cshtml]: @andymantell well spotted Andy, I have removed it.

File: Frontend/CO.CDP.OrganisationApp/Pages/Forms/_FormElementTextInput.cshtml (hunk 2 above)
Comment (JBaigGoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Forms/_FormElementTextInput.cshtml]: @andymantell well spotted Andy, I have removed it.

File: Frontend/CO.CDP.OrganisationApp/Pages/Forms/_FormElementTextInput.cshtml [hunk 3]
```diff
@@ -5,14 +5,14 @@
 }
 
 <div class="govuk-form-group">
-    @if (!string.IsNullOrWhiteSpace(Model.Caption))
+   
+    @if (!string.IsNullOrWhiteSpace(Model.Heading))
     {
-        <span class="govuk-caption-l">@Model.Caption</span>
+        <h1 class="govuk-label govuk-label--l" for="textInput">@Model.Heading</h1>
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Forms/_FormElementTextInput.cshtml:11]:
```suggestion
        <h1 class="govuk-label-wrapper">
            <label class="govuk-label govuk-label--l" for="textInput">@Model.Heading</label>
        </h1>
```

File: Frontend/CO.CDP.OrganisationApp/Pages/Forms/_FormElementMultiLineInput.cshtml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+@{
+    var hasError = ((TagBuilder)Html.ValidationMessageFor(m => m.TextInput)).HasInnerHtml;
+}
+
+<div class="govuk-form-group">
+
+    @if (!string.IsNullOrWhiteSpace(Model.Heading))
+    {
+        <h1 class="govuk-label govuk-label--l" for="multiLineTextInput">@Model.Heading</h1>
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Forms/_FormElementMultiLineInput.cshtml:11]:
```suggestion
        <h1 class="govuk-label-wrapper">
            <label class="govuk-label govuk-label--l" for="multiLineTextInput">@Model.Heading</label>
        </h1>
```

**Review by andymantell — CHANGES_REQUESTED** [review]

**Review by andymantell — APPROVED** [review]

**Pull request merged**